src/OSPSuite.Core/Domain/ContainerExtensions.cs

```diff
@@ -51,16 +51,6 @@ public static IEnumerable<IParameter> AllParameters(this IContainer container, F
          return container;
       }
 
-      internal static IEnumerable<IContainer> AllContainersFor(this IEnumerable<IContainer> containers, DescriptorCriteria containerCriteria)
```

Comment:
Those methods were internal and not used. They now go against what I have implemented so I removed them